data_uploader/netbox_api/netbox_connection.py

```diff
@@ -0,0 +1,33 @@
+from data_uploader.exceptions.missing_mandatory_param_error import MissingMandatoryParamError
+import requests
+from pynetbox import api
+
+
+class NetboxApi:
+    """
+    Wraps a netbox connection as a context manager
+
+    """
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def api_object(netbox_url: str, token: str, cert=None):
+        if not netbox_url:
+            raise MissingMandatoryParamError(
+                "NetBox URL is required but not provided."
+            )
+        if not token:
+            raise MissingMandatoryParamError(
+                "A token is required but not provided."
+            )
+        session = requests.Session()
+        if not cert:
+            session.verify = False
+        else:
+            session.verify = cert
```

Comment:
Nit, I would do this in a single line

`session.verify = cert if cert else False`